docs/customization/controller.rst

```diff
@@ -1,25 +1,26 @@
 Customizing Controllers
 =======================
 
-All **Sylius** resources are using the
+All **Sylius** resources use the
 `Sylius/Bundle/ResourceBundle/Controller/ResourceController <https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php>`_
-as default, but some of them have been already extended in Bundles.
-If you want to override some controller action, check which controller you should be extending.
+by default, but some of them have already been extended in Bundles.
+If you want to override a controller action, check which controller you should be extending.
 
 .. note::
-    There are two types of controllers we can define in Sylius.
-    **Resource Controllers** - are basing only on one Entity, so they return only the resources they have in their name. For instance a ``ProductController`` should return only products.
+    There are two types of controllers we can define in Sylius:
+
+    **Resource Controllers** - are based only on one Entity, so they return only the resources they have in their name. For instance a ``ProductController`` should return only products.
     **Standard Controllers** - non-resource; these may use many entities at once, they are useful on more general pages. We are extending these controllers only if the actions we want cannot be done through yaml configuration - like sending emails.
 
 Why would you customize a Controller?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To add your custom actions you need to override controllers. You may bee needing to:
+To add your custom actions you need to override controllers. You may need to:
 
-* add a generic action that will render a list of recommended products with a product on its show page,
+* add a generic action that will render a list of recommended products with a product on its show page.
 * render a partial template that cannot be done via yaml resource action.
 
-How to customize a Resource Controller?
+How to customize a Resource Controller:
```

Comment:
No problem! How should I update the pull request, do I submit a new one without the changes you've mentioned?